notebooks/vertex_genai/solutions/vertex_llm_tuning.ipynb

```diff
@@ -0,0 +1,1001 @@
+{
```

Comment:
<p>Maybe better to include the expected training time.</p><p>(More than a couple of hours?)</p><p> </p><p> </p><p>Also, adding an info that the job runs on Vertex AI Pipelines would be helpful.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/395/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>Good catch! For 100 steps, it's 1h. For 500, it's 3:30 for me. Done!</p>